app/src/main/java/org/cru/godtools/ui/dashboard/tools/ToolCategoriesAdapter.kt

```diff
@@ -14,21 +14,21 @@ class ToolCategoriesAdapter(
     lifecycleOwner: LifecycleOwner,
     private val selectedCategory: LiveData<String?>,
     private val primaryLanguage: LiveData<Locale>
-) : SimpleDataBindingAdapter<DashboardToolsCategoriesCategoryBinding>(lifecycleOwner), Observer<List<String>> {
+) : SimpleDataBindingAdapter<DashboardToolsCategoriesCategoryBinding>(lifecycleOwner), Observer<List<String?>> {
     interface Callbacks {
         fun onCategorySelected(category: String?)
     }
 
     val callbacks = ObservableField<Callbacks>()
-    private var categories = emptyList<String>()
+    private var categories = emptyList<String?>()
         set(value) {
             field = value
             notifyDataSetChanged()
         }
 
-    override fun getItemCount() = categories.size
+    override fun getItemCount() = if (categories.isNotEmpty()) categories.size + 1 else 0
```

Comment:
this could be simplified to `categories.size + 1`. the categories RecyclerView won't actually be visible when there are no categories.